AK/Span.h

```diff
@@ -156,6 +156,10 @@ class Span : public Detail::Span<T> {
     ALWAYS_INLINE constexpr size_t copy_trimmed_to(Span<RemoveConst<T>> other) const
     {
         auto const count = min(size(), other.size());
+        if (count == 1) {
```

Comment:
This is wrong if `T` is not a trivial type, rogt

Reply:
That could be the case, as it doesn't improve much anyways, I'll drop it.